fs/littlefs/Kconfig

```diff
@@ -129,6 +129,18 @@ config FS_LITTLEFS_LOCAL_PATCHES
 		patches for LITTLEFS.
 		This is NOT expected to be tweaked directly by users.
 
+config FS_LITTLEFS_HAS_LFS_DEFINES
```

Comment:
why need option? it's better to check version and apply the new method automatically

Reply:
how to check the version?

Reply:
compare CONFIG_FS_LITTLEFS_VERSION in Makefile/CMakeList.txt) directly.

Reply:
such a comparison in Makefile/CMakeList.txt is too complex to implement
and won't work for arbitrary tags and git sha1.

Reply:
OK